package.json

```diff
@@ -65,15 +65,17 @@
     "postcss-for": "^2.1.1",
     "postcss-mixins": "^6.0.0",
     "prop-types": "^15.5.10",
+    "query-string": "^5.0.0",
     "rally-point-player": "^1.0.0",
     "react": "^15.4.1",
     "react-addons-css-transition-group": "^15.4.1",
     "react-custom-scrollbars": "^4.0.0",
     "react-dom": "^15.4.1",
     "react-ga": "^2.1.2",
     "react-redux": "^5.0.1",
-    "react-router": "^2.6.0",
-    "react-router-redux": "^4.0.5",
+    "react-router": "^4.2.0",
```

Comment:
Note to self: Can remove this dependency since we're using `react-router-dom` to import things.